### PR TITLE
Updated aws-ec2-inventory docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md [hunk 1]
```diff
@@ -44,10 +62,13 @@ To configure a CSE AWS EC2 Inventory Source:
...
+1. **AWS Access**. The integration is configured for either role based AWS authentication or key based AWS authentication.
+    - **Role Based Access**. AWS Role ARN is required for Role based Access. Use the information provided on the source page to configure the role.<br/><img src={useBaseUrl('/img/send-data/role-based.png')} alt="role-based" style={{border: '1px solid black'}} width="400"/>
+    - **Key Access**. Enter the IAM user access key ID and secret key you want to use to authenticate collection requests.<br/><img src={useBaseUrl('/img/send-data/key-based.png')} alt="key-based" style={{border: '1px solid black'}} width="400"/>
 1. **Regions**. Provide a list of AWS regions to query EC2 instances, such as `us-east-2`.
 1. (Optional) The **Polling Interval** is set for 600 minutes by default, you can adjust it based on your needs.
-1. When you are finished configuring the Source, click **Submit**.
+1. **Processing Rules for Logs (Optional)**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in Create a Processing Rule.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md:70]:
```suggestion
1. **Processing Rules for Logs (Optional)**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule/).
```

**Review by jpipkin1 — APPROVED** [review]
@JV0812, I had only one comment. I approved on the assumption you'll make that change.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md [hunk 2]
```diff
@@ -26,6 +26,24 @@ The table below shows the AWS source fields that CSE maps to CSE schema attribut
 
 The [IAM policy](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-policies-for-amazon-ec2.html) needs the `ec2:DescribeInstances` and `ec2:DescribeImages` permissions.
 
+## States
+
+A CSE AWS EC2 Inventory Source tracks errors, reports its health, and start-up progress. You’re informed, in real-time, if the Source is having trouble connecting, if there's an error requiring user action, or if it is healthy and collecting by utilizing Health Events.
+
+A CSE AWS EC2 Inventory Source goes through the following states when created:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md:33]:
```suggestion
The CSE AWS EC2 Inventory Source tracks errors, reports its health, and start-up progress. You’re informed, in real-time, if the Source is having trouble connecting, if there's an error requiring user action, or if it is healthy and collecting by utilizing Health Events.

The CSE AWS EC2 Inventory Source goes through the following states when created:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md [hunk 3]
```diff
@@ -69,9 +90,11 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
 
 | Parameter | Type | Required | Description | Access |
 |:--|:--|:--|:--|:--|
-| config | JSON Object | Yes | Contains the configuration parameters for the Source. |   |
+| config | JSON Object | Yes | Contains the configuration parameters for the Source. | na |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md:93]:
```suggestion
| config | JSON Object | Yes | Contains the configuration parameters for the Source. | n/a |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md [hunk 4]
```diff
@@ -81,27 +104,31 @@ The following table shows the ****config**** parameters.
...
+|`authentication.roleARN` | String | Yes | |Provide the IAM Role ARN you want to use to authenticate collection requests. | modifiable |
+|`authentication.awsId` | String | Yes | | Provide the IAM User access key ID you want to use to authenticate collection requests. | modifiable |
+|`authentication.roleARN` | String | Yes | |Provide the Secret Key you want to use to authenticate collection requests. | modifiable |
+|`polling_interval` | Integer | No | 600 | The minutes in between checks for new data. | modifiable |
 
-CSE AWS EC2 Inventory Source JSON example:
+### JSON Example
 
-```json
+```json title="JSON example for Role Based Authentication"
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cse-aws-ec-inventory-source.md:116]:
```suggestion
```json title="JSON example for Role-Based Authentication"
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor formatting

**Review by kimsauce — APPROVED** [review]

**Pull request merged**